src/user-guide/sharing-workflows/sharing-workflows.rst

```diff
@@ -0,0 +1,285 @@
+.. _Authorization:
+
+Authorization
+=============
+
+Cylc supports multi user access. You can grant control of your
+workflows to other users, by adding or removing privileges on a Cylc operation
+basis.This requires site level configuration, with individual users being able
+to set authorization settings, within the bounds set by the site.
+
+Sites can set default access rights for users, for more information, see
+:ref:`site_configuration`.
+
+Note, if this feature is not configured, the default is no multi user access
+and access will be limited to the workflow owner.
+
+If you grant access to another user, this access will apply to all workflows.
+
+Granting Access
+---------------
+
+There are three methods of identifying a user to grant access to:
+
+- ``*`` character, to indicate any authenticated user,
+- using the ``group:`` prefix to indicate a system group. E.g.
+  ``group:groupname``, will assign permissions to all members of the system
+  group named `groupname`. For more information, see :ref:`group_support`.
+- ``username`` to indicate a specific user.
+
+All permissions are additive, if user appears elsewhere in configuration, for
+example as a member of a system group, the permission level is taken as the
+greatest possible.
+
+However, any negations are applied with priority.
+
+Note, default in site config does not contribute to permissions, this is used
+in the case where a user does not appear in the `c.CylcUIServer.user-authorization`.
+
+.. note::
+
+   Spellings of UIServer configurations should be strictly adhered to. Using,
+   for example, `c.CylcUIServer.user-authorisation` rather than
+   `c.CylcUIServer.user-authorization` would result in cylc silently ignoring
+   this configuration.
+
+Methods of Assigning Permissions
+--------------------------------
+Assigning permissions can be done in two ways:
+
+ - using individual Cylc operations, e.g. ``play``, ``pause``
+ - using predefined access groups: ``READ``, ``CONTROL``, ``ALL``.
+
+Using both methods is supported, e.g ["READ", "stop", "pause"]
+
+
+Individual Operations
+^^^^^^^^^^^^^^^^^^^^^ 
+To assign users permissions, you can list the operations you wish to grant
+
+.. code-block:: python
+
+  c.CylcUIServer.user_authorization = {
+      "user1": ["read", "pause", "play"]
+      }
+
+Provided you have permission to give `user1` these permissions, as defined in the
+site configuration file, this will result in `user1` being able to see your
+workflows (from the `read` operation), and `pause` and `play` your workflows.
```

Comment:
Single backquotes have meaning in RST, can use double backquotes for literals or regular quotes otherwise.

Will need to check use of single backquotes throughout this page.